### DISCUSSION TITLE
No firewall configuration option to allow Desktop

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#13376] [2.4] [answered]
### Version

2.4.80

### Installation Method

Security Onion ISO image

### Description

configuration

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

100

### Storage for /nsm

1000

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When installing a SO Desktop from the ISO, at the end it tries to connect to the grid.  When this fails, it prompts you with the command to manually add the desktop to firewall, and also tells you that you can go Administration->Configuration and use one of the firewall options to add the Desktop.
Desktop is not one of the options in the list of Allow actions
Running the so-firewall command works ok

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by cm-ops** [comment, marked as answer]
Did you select the advanced toggle in SOC? `Administration > Configuration > firewall > hostgroups > desktop`

![image](https://github.com/user-attachments/assets/35c35bb4-7a0e-4f27-a68b-7553491157cb)

**Comment by rosswakelin** [comment]
No.  I was trying to follow the instruction given in the Desktop installation popup, which pointed me to an option that did not exist.  This discussion was just to recommend that the popup gets fixed, or the option gets added where the popup says there is an option.

**Reply by dougburks** [reply]
@rosswakelin To make sure we understand the issue here, are you referring to this message in Setup?
```
Setup is unable to access the manager. This most likely means that you need to allow this machine to connect through the manager's firewall. 

You can either go to SOC --> Administration --> Configuration and choose the correct firewall option from the list OR you can run the following command on the manager:
```

When I go to SOC --> Administration --> Configuration and drill down into `firewall`, I can select `desktop` from the list:
![image](https://github.com/user-attachments/assets/d2ab97c1-70e6-4dcc-8253-c71bb60fb3cb)

Or are you referring to the firewall quick links on the right side?
![image](https://github.com/user-attachments/assets/820541e4-a4c8-4a75-b7a6-8a7765cbc2b7)

We don't create quick links for every possible firewall option, but perhaps we could add one for Security Onion Desktop if that would help.

**Reply by rosswakelin** [reply]
Morning

Yes I was referring to the quicklinks.  Just trying to make the interface consistent for users.

**Reply by dougburks** [reply]
@rosswakelin I've created issue https://github.com/Security-Onion-Solutions/securityonion/issues/13412 for this.
